packages/web-react/src/types/link.ts

```diff
@@ -17,7 +25,9 @@ export interface LinkBaseProps<C = void> extends ChildrenProps, StyleProps, Tran
   /** Color of the Link */
   color?: ActionLinkColorsDictionaryType<C>;
   /** Whether is the Link underlined */
+  /** @deprecated "isUnderlined" property will be replaced in the next major version. Please use "underline" instead. */
```

Comment:
Do we have an issue for a codemod? Or shouldn't we create it already in this PR?

Reply:
Added new codemod for this change